tests/unittests/core/io/test_cmdline_parser.py

```diff
@@ -120,6 +119,7 @@ def test_parse_arguments_bad_command():
 
 
 def test_parse_branching_arguments_template():
+    """Test the branching of args"""
```

Comment:
This we should not support, it is kleio specific. You could drop this unit-test.